py-polars/polars/_utils/parse_expr_input.py

```diff
@@ -121,6 +121,12 @@ def parse_as_expression(
     elif isinstance(input, list) and not list_as_lit:
         expr = F.lit(pl.Series(input), dtype=dtype)
         structify = False
+    elif isinstance(input, pl.DataFrame):
+        msg = "A DataFrame is an invalid input"
+        raise InvalidOperationError(msg)
+    elif isinstance(input, pl.LazyFrame):
+        msg = "A LazyFrame is an invalid input"
```

Comment:
```suggestion
        msg = "a LazyFrame is an invalid input"
```

We don't want to capitalize the first line of an error message.